libs/symphonia/src/testing/mod.rs

```diff
@@ -113,3 +150,35 @@ pub struct MockNet {
 ///
 /// Elements can be stuffed in, and they will be delayed until a given time or lost before they can be read out. This simulates a bad network connection or other similar construct.
 pub struct LossyChan;
+
+enum Event {
+    Sent {
+        sender: Ed25519PK,
+        destination: Ed25519PK,
+        content: String,
+    },
+    Received {
+        sender: Ed25519PK,
+        destination: Ed25519PK,
+        content: String,
```

Comment:
@nullchinchilla is content supposed to be a string for metrics gathering purposes?